crates/rpc/rpc-eth-types/Cargo.toml

```diff
@@ -27,13 +27,15 @@ reth-rpc-types-compat.workspace = true
 reth-tasks.workspace = true
 reth-transaction-pool.workspace = true
 reth-trie.workspace = true
+reth-provider.workspace = true
```

Comment:
do we really need this or do we only need certain traits? (storage-api)

Reply:
is it better to import
```rust
use reth_storage_api::BlockReaderIdExt;
use reth_chain_state::CanonStateSubscriptions;
use reth_chainspec::ChainSpecProvider;
```
?

Reply:
yes, the provider crate is pretty beefy